spec/javascripts/govuk_publishing_components/analytics-ga4/ga4-link-tracker.spec.js

```diff
@@ -452,5 +452,23 @@ describe('GA4 link tracker', function () {
         expect(window.dataLayer[0].event_data.text).toEqual('image')
       }
     })
+
+    it('sets the text property to image when the nested elements are clicked', function () {
+      element = document.createElement('div')
+      element.setAttribute('data-ga4-track-links-only', '')
+      element.setAttribute('data-ga4-link', '{"someData": "blah"}')
+      element.innerHTML = '<a href="#link1"><img class="link" src=""/></a>' +
+      '<a href="#link1"><svg class="link"></svg></a>' +
+      '<a href="#link1"><svg><path class="link"></path></svg></a>'
```

Comment:
Should this also include a test link where the link has been clicked, i.e.

```html
<a href="#" class="link"><img src=""/></a>
```

Reply:
The previous tests above this test should cover that :+1:

Reply:
Great, thanks.